ngx-fudis/projects/ngx-fudis/src/lib/components/dialog/ngMaterial-theme.scss

```diff
@@ -91,6 +95,16 @@
             padding: spacing.$spacing-none;
           }
         }
+
+        & fudis-form {
+          & .fudis-grid__xxl {
+            @include breakpoints.breakpoint('sm') {
+              @include grid.grid-max-width('xxl');
+
+              gap: spacing.$spacing-sm;
+            }
```

Comment:
I'd like to see and understand reasoning behind this! I understand that for the "first level" grid this is okay, but what if Form has nested grids? This would affect those as well. Is that intended?

Reply:
The reason behind this is that we want form action buttons to always flow to dialog's right corner when dialog is enlargened. Without this style, form within a dialog would have buttons hanging in a middle of large dialogs and this is not designer's intention. The default gap size also led to an unnecessary increase in dialog height.

Reply:
Discussed and renamed/refactored these